README.md

```diff
@@ -41,9 +41,10 @@ touch ~/.netrc #prevents docker compose from creating it as a directory if you d
 
 docker-compose run web bin/rake db:setup
 
-# To get a copy of the Prod database as test data.
+# Run this to get a copy of the Prod database as test data.
+# Enter `password` when prompted for a password after the download step.
+# The restore step after that will take a very long time. Hours maybe.
```

Comment:
Hmmm, it shouldn't take hours to restore. It usually finishes in 5 minutes or so for me. If it's taking hours, that's odd.